### ISSUE TITLE
Request for details on accuracy validation

### ISSUE THREAD

**Issue by banafshebamdad** [facebookresearch/projectaria_tools#153]
Hello, 
I am using the closed-loop trajectories generated by MPS SLAM service as ground truth for evaluating SLAM systems. The Project Aria paper reports a typical global RMSE translation error of ≤1.5 cm in room-scale scenarios. I would greatly appreciate any information you could share regarding the validation methodology used to establish the reported accuracy specifications
I understand that some details may be proprietary, and any information you can share would be invaluable.
Best regards,
Banafshe

**Comment by SeaOtocinclus** [comment]
Thank you for your question @banafshebamdad 

We observe 0.1 to 0.2 % positional drift error for open loop tracking, most of it due to scale error that we evaluated on proprietary dataset. This error is translating to centimeter accuracy (1.5cm over 10meter -- a room scale environment).

We also evaluated with MOCAP ground-truth data such as the [ADT Dataset](https://www.projectaria.com/datasets/adt/) and see that our error range is comparable to the tracking MOCAP system.

Do note that close loop trajectory should be as good or better than open loop trajectories.

Final note: Accuracy can vary depending on user motion pattern and environments (adversarial motion and featureless environment can make SLAM harder).
